keymaps/win32.cson

```diff
@@ -67,10 +67,10 @@
   'ctrl-_': 'window:decrease-font-size'
   'ctrl-0': 'window:reset-font-size'
 
-  'ctrl-k up': 'pane:split-up' # Atom Specific
-  'ctrl-k down': 'pane:split-down' # Atom Specific
-  'ctrl-k left': 'pane:split-left' # Atom Specific
-  'ctrl-k right': 'pane:split-right' # Atom Specific
+  'ctrl-k up': 'window:copy-item-to-pane-above' # Atom Specific
```

Comment:
Maybe this should this be more explicit at the cost of being a bit longer... `copy-active-item-to-pane-above`. @maxbrunsfeld what do you think?